core/common/src/main/java/alluxio/util/executor/ExecutorServiceFactories.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.util.executor;
+
+import alluxio.util.ThreadFactoryUtils;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * Convenience methods for constructing instances of {@link ExecutorServiceFactory}.
+ */
+public final class ExecutorServiceFactories {
```

Comment:
It's convenient for avoiding messy syntax anywhere that we need an `ExecutorServiceFactory`.